content_templates/templates/common/instance_view_builder.html

```diff
@@ -0,0 +1,405 @@
+      }
+      union {
+        ?instance sh:hasValue ?value
+      }
+      union {
+        ?instance sh:minCount ?minCount
+      }
+      union {
+        ?instance lam:hasAnnotationConfiguration ?configurations
+      }
+    }" %}
+    {% set content, error = from_endpoint(conf.default_endpoint).with_query(select_query).fetch_tabular() %}
+    {% call macros.render_fetch_results(content, error) %}
+        {% if not content.dropna(how="all").empty %}
+            {{ macros.render_description_item_local_iri (content, "path", "Path") }}
+            {{ macros.render_description_item_plain_literal (content, "class", "Class") }}
+            {{ macros.render_description_item_plain_literal (content, "value", "Value") }}
+            {{ macros.render_description_item_plain_literal (content, "minCount", "Min Count") }}
+            {{ macros.render_description_item_plain_literal (content, "maxCount", "Max Count") }}
+            {% set values =  content["configurations"].dropna() %}
+            {% if not values.empty %}
+                <p class="debug">build_lam_property_configuration for hasAnnotationConfiguration
+                    <strong>{{ instance_uri }}</strong></p>
+                {% for _, item in values.iteritems() %}
+                    {{ build_lam_property_configuration(graph, item, level + 1) }}
+                {% endfor %}
+            {% endif %}
+        {% endif %}
+    {% endcall %}
+{% endmacro %}
+
+
+{% macro build_lam_classification_hint_list(graph, instance_uri, level) %}
+    <p class="debug">build_lam_property_configuration_list <strong>{{ instance_uri }}</strong></p>
+    {% set select_query =
+    "PREFIX sh: <http://www.w3.org/ns/shacl#>
+    PREFIX lam: <http://publications.europa.eu/ontology/lam-skos-ap#>
+
+    select ?classificationHint
+    FROM <"~graph~">
+    {
+      values ?instance {<"~instance_uri~">}
+      ?instance lam:classifyWith ?classificationHint .
+    }" %}
+    {% set content, error = from_endpoint(conf.default_endpoint).with_query(select_query).fetch_tabular() %}
+    {% call macros.render_fetch_results(content, error) %}
+        {% for row in content.itertuples() %}
+            {{ build_lam_classification_hint(graph, row.classificationHint, level) }}
+        {% endfor %}
+    {% endcall %}
+{% endmacro %}
+
+
+{% macro build_lam_classification_hint(graph, instance_uri, level) %}
+    <p class="debug">build_lam_classification_hint <strong>{{ instance_uri }}</strong></p>
+
+    {% set select_query =
+    "PREFIX sh: <http://www.w3.org/ns/shacl#>
+
+    select ?name
+    FROM <"~graph~">
+    {
+      values ?instance {<"~instance_uri~">}
+      ?instance sh:name ?name .
+    }" %}
+    {% set content, error = from_endpoint(conf.default_endpoint).with_query(select_query).fetch_tabular() %}
+    {% call macros.render_fetch_results(content, error) %}
+        {% if not content.empty %}
+            {% for row in content.itertuples() %}
+                <h{{ level }}>
```

Comment:
please use Headings for Classes and Collections; anything else inside the class description (even if an object) shall be rendered as a section or list item.